src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const displayName = 'dashboard.ActionsPage.MintTokenMotion';
+
+const MintTokenMotion = () => {
+  return <div />;
+};
+
+MintTokenMotion.displayName = displayName;
+
+export default MintTokenMotion;
```

Comment:
This needs a bit more content:
- header title
- annotation hash
- details widget

Have a look at the `RecoveryAction`  component and copy them from there _(adapted to this motion, of course)_